students/JugalKishore/Session_3/list.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+# Series 1
+print("####Series 1####")
+print()
+
+# Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”
+list = ['Apples', 'Pears', 'Oranges', 'Peaches']
```

Comment:
list is a built in name in Python - you'll want to avoid using this as an actual list name.  Better to use fruit_list or the like.